docs/en/transform-v2/common-options.md

```diff
@@ -1,23 +1,65 @@
+| result_table_name | String | No       | -       | When `source_table_name` is not specified, the current plug-in processes the data set `(dataset)` output by the previous plug-in in the configuration file; <br/>When `source_table_name` is specified, the current plugin is processing the data set corresponding to this parameter.                                                                                                                                                                                                                                                |
+| source_table_name | String | No       | -       | When `result_table_name` is not specified, the data processed by this plugin will not be registered as a data set that can be directly accessed by other plugins, or called a temporary table `(table)`; <br/>When `result_table_name` is specified, the data processed by this plugin will be registered as a data set `(dataset)` that can be directly accessed by other plugins, or called a temporary table `(table)` . The dataset registered here can be directly accessed by other plugins by specifying `source_table_name` . |
 
-### source_table_name [string]
+## Task Example
 
-When `source_table_name` is not specified, the current plug-in processes the data set `(dataset)` output by the previous plug-in in the configuration file;
+### Simple:
 
-When `source_table_name` is specified, the current plugin is processing the data set corresponding to this parameter.
+> This is the process of converting the data source to fake and write it to two different sinks, Detailed reference `transform`
 
-### result_table_name [string]
+```bash
+env {
+  job.mode = "BATCH"
+}
 
-When `result_table_name` is not specified, the data processed by this plugin will not be registered as a data set that can be directly accessed by other plugins, or called a temporary table `(table)`;
+source {
+  FakeSource {
+    result_table_name = "fake"
+    row.num = 100
+    schema = {
+      fields {
+        id = "int"
+        name = "string"
+        age = "int"
+        c_timestamp = "timestamp"
+        c_date = "date"
+        c_map = "map<string, string>"
+        c_array = "array<int>"
+        c_decimal = "decimal(30, 8)"
+        c_row = {
+          c_row = {
+            c_int = int
+          }
+        }
+      }
+    }
+  }
+}
 
-When `result_table_name` is specified, the data processed by this plugin will be registered as a data set `(dataset)` that can be directly accessed by other plugins, or called a temporary table `(table)` . The dataset registered here can be directly accessed by other plugins by specifying `source_table_name` .
+transform {
```

Comment:
Need add a example for Multiple Tables.

Reply:
> Need add a example for Multiple Tables.

@EricJoy2048 Do you have a specific reference case let me add